### PR TITLE
Create jesslyn-difiori

### PR DESCRIPTION
**Pull request by jkdifiori**
created bio

### CODE REVIEW

**Review by agentrrb — APPROVED** [review]

File: team/jesslyn-difiori [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Jesslyn Di Fiori
+  avatar: https://avatars.githubusercontent.com/jkdifiori
+  role: Technical Writer
+  organizations:
+    - name: National Snow and Ice Data Center (NSIDC)
+  bio: >
+    A FEW SENTENCES ABOUT YOU, MULTIPLE LINES ARE FINE
```
Comment (scottyhq) [inline team/jesslyn-difiori]: thanks @jkdifiori don't forget to modify these other sections in CAPS before we can merge your bio

**Review by scottyhq — CHANGES_REQUESTED** [review]

File: team/jesslyn-difiori [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Jesslyn Di Fiori
+  avatar: https://avatars.githubusercontent.com/jkdifiori
+  role: Technical Writer
+  organizations:
+    - name: National Snow and Ice Data Center (NSIDC)
+  bio: >
+    Jesslyn is a Technical Writer at the National Snow and Ice Data Center, where she writes and maintains scientific documentation, metadata, and web content for cryospheric data. She works primarily with NASA SnowEx, MODIS, and VIIRS data sets. 
+    - Scientific Communication
+    - Data Visualization
```
Comment (scottyhq) [inline team/jesslyn-difiori:10]:
```suggestion
expertise:
    - Scientific Communication
    - Data Visualization
```

File: team/jesslyn-difiori [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Jesslyn Di Fiori
+  avatar: https://avatars.githubusercontent.com/jkdifiori
+  role: Technical Writer
+  organizations:
+    - name: National Snow and Ice Data Center (NSIDC)
+  bio: >
+    Jesslyn is a Technical Writer at the National Snow and Ice Data Center, where she writes and maintains scientific documentation, metadata, and web content for cryospheric data. She works primarily with NASA SnowEx, MODIS, and VIIRS data sets. 
+    - Scientific Communication
```
Comment (scottyhq) [inline team/jesslyn-difiori]:
```suggestion
  bio: >
    Jesslyn is a Technical Writer at the National Snow and Ice Data Center, where she writes and maintains scientific documentation, metadata, and web content for cryospheric data. She works primarily with NASA SnowEx, MODIS, and VIIRS data sets. 
  expertise:
    - Scientific Communication
```

Comment (scottyhq) [reply team/jesslyn-difiori]: @jkdifiori see suggestion above to add back missing 'expertise:' header

File: team/jesslyn-difiori [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+- title: Jesslyn Di Fiori
+  avatar: https://avatars.githubusercontent.com/jkdifiori
+  role: Technical Writer
+  organizations:
+    - name: National Snow and Ice Data Center (NSIDC)
+  bio: >
+    Jesslyn is a Technical Writer at the National Snow and Ice Data Center, where she writes and maintains scientific documentation, metadata, and web content for cryospheric data. She works primarily with NASA SnowEx, MODIS, and VIIRS data sets. 
+  expertise:     
+- Scientific Communication
+    - Data Visualization
```
Comment (scottyhq) [inline team/jesslyn-difiori]:
```suggestion
  expertise:     
    - Scientific Communication
    - Data Visualization
```

be sure to match indentation levels

Comment (scottyhq) [reply team/jesslyn-difiori]: (if you click the 'commit suggestion' button above should be all set. the formatting is very picky!)

**Comment by scottyhq** [comment]
@jkdifiori sorry I missed this earlier. You'll also need to rename your file `jesslyn-difiori` -> `jesslyn-difiori.yaml`

**Review by scottyhq — APPROVED** [review]

**Comment by scottyhq** [comment]
thanks!

**Pull request merged**